app/views/admin/events/index.csv.haml

```diff
@@ -0,0 +1,14 @@
+- headers = ['Title',
+             'Abstract',
+             'Start time',
+             'Submitter',
+             'Speaker',
+             'Event Type',
+             'Track',
+             'Difficulty Level',
+             'State']
+= CSV.generate_line headers
+- @events.each do |event|
+  = CSV.generate_line([event.title, event.abstract, event.start_time,
```

Comment:
No, not really. What an organizer would really care about is for the start_time of the selected_scheduled (ie. the time when the event is actually taking place).